portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,13 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+
+window.onscroll = function() {myFunction()};
+
+function myFunction() {
```

Comment:
You can rename myFunction to something a bit clearer e.g. onScroll, handleOnScroll, etc.

Reply:
Fixed, thank you!